src/worker.js

```diff
@@ -45,9 +45,9 @@ Module['instantiateWasm'] = function(info, receiveInstance) {
   // Instantiate from the module posted from the main thread.
   // We can just use sync instantiation in the worker.
   var instance = new WebAssembly.Instance(Module['wasmModule'], info);
+  receiveInstance(instance, Module['wasmModule']);
```

Comment:
How about ifdefing here to avoid always passing `Module['wasmModule']`? we only need it when RELOCATABLE, and this is something I doubt closure will eliminate (but maybe worth checking)

Reply:
OK.. seems a little excessive to me and I guess should also update the other callsites too though for consistency?

Reply:
Done